resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md

```diff
@@ -0,0 +1,315 @@
+#import "ACPIdentity.h"
+
+- (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [ACPCore setLogLevel:ACPMobileLogLevelDebug];
+    [ACPAnalytics registerExtension];
+    [ACPIdentity registerExtension];
+    [ACPCore start:^{
+      // add your app id from the "Environments" tab on Launch.
+  		[ACPCore configureWithAppId:@"your-app-id"];
+    }];
+    return YES;
+}
+```
+
+**Swift**
+
+```swift
+import ACPCore
+import ACPAnalytics
+import ACPIdentity
+
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+  ACPCore.setLogLevel(ACPMobileLogLevel.debug)
+  ACPAnalytics.registerExtension()
+  ACPIdentity.registerExtension()
+  ACPCore.start(){
+  	ACPCore.configureWithAppId("your-app-id") 
+  }
+  return true
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+In-depth instructions can be seen at the [Analytics Readme](../../using-mobile-extensions/adobe-analytics/readme.md#add-analytics-to-your-app).
+
+##### Track App State and Track App Actions
+
+The V4 syntax and usage examples for these API are:
+
+{% tabs %}
+{% tab title="Android" %}
+
+```java
+// syntax
+public static void trackState(final String state, final Map<String, Object> contextData)
+
+// usage
+Analytics.trackState("MainPage", new HashMap<String, Object>() {{
+  put("firstVisit", true);
+}});
+```
+
+```java
+// syntax
+public static void trackAction(final String action, final Map<String, Object> contextData)
+  
+// usage
+Analytics.trackAction("linkClicked", new HashMap<String, Object>() {{
+  put("url", "https://www.adobe.com");
+}});
+```
+
+The AEP SDK's have moved the `trackAction` and `trackState` APIs to the MobileCore extension. In addition, the context data Map has been changed from `<String, Object>` to `<String, String>`: 
+
+```java
+// syntax
+public static void trackState(final String state, final Map<String, String> contextData)
+  
+// usage
+MobileCore.trackState("MainPage", new HashMap<String, String>() {{
+  put("firstVisit", "true");
+}});
+```
+
+```java
+// syntax
+public static void trackAction(final String action, final Map<String, String> contextData)
+  
+// usage
+MobileCore.trackAction("linkClicked", new HashMap<String, String>() {{
+  put("url", "https://www.adobe.com");
+}});
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+```objective-c
+// syntax
++ (void) trackState:(NSString *)state data:(NSDictionary *)data;
+
+// usage
+[ADBMobile trackState:@"MainPage" data:@{@"firstVisit":@true}];
+```
+
+```objective-c
+// syntax
++ (void) trackAction:(NSString *)action data:(NSDictionary *)data;
+
+// usage
+[ADBMobile trackAction:@"linkClicked" data:@{@"url":@"https://www.adobe.com"}];
+```
+
+The AEP SDK's have moved the `trackAction` and `trackState` API's to the MobileCore extension. In addition, the NSDictionary has been changed from `<NSString, NSObject>` to `<NSString, NSString>`: 
```

Comment:
Feel like need to emphasis this section for AEP SDK implementation so user can look at this directly.  May be bold text or highlight?